README.md

```diff
@@ -40,6 +40,7 @@ To run the app, follow these steps.
   gulp watch
   ```
 7. Browse to [http://localhost:9000](http://localhost:9000) to see the app. You can make changes in the code found under `src` and the browser should auto-refresh itself as you save files.
+> The Skeleton App uses [BrowserSync](http://www.browsersync.io/) for automated page refreshes on code/markup changes concurrently accross multiple browsers. If you prefer to disable the mirroring feature set the [ghostMode option](http://www.browsersync.io/docs/options/#option-ghostMode) to false
```

Comment:
Ah yes true. I wasn't paying attention and thought this was the tutorial. I will look at making a PR on the tutorial itself.